docs/BgpConfigDeploymentType.md

```diff
@@ -0,0 +1,13 @@
+# BgpConfigDeploymentType
+
+## Enum
+
+
+* `GLOBAL` (value: `"global"`)
```

Comment:
Looks like the `enumClassnamePrefix` option isn't used in the docs, so the enum names here are wrong, but IMO that's something we can accept for now and propose a fix upstream.  Alternative would be to eject the docs template and customize it but that feels unnecessary here.